src/components/DashboardResultIndicatorsSection.vue

```diff
@@ -251,8 +240,8 @@ export default {
       immediate: true,
       async handler() {
         this.latestProgressRecord = null;
-
         this.getProgressData().then(this.renderGraph);
+        this.getGoal().then((goal) => this.goal = goal);
```

Comment:
I og med at vi kun henter mål ved skifte av fane, blir målet stående for ny RI dersom en RI-en man "står på" degraderes til en annen type. Dette er vel et "kanttilfelle", men om man f.eks. har dashbordet oppe på en skjerm, er det kanskje mer relevant (ref punktet om binding).
<img width="276" alt="image" src="https://user-images.githubusercontent.com/2866225/201067014-c5a0336e-918e-4124-b920-6dfe1ae7f9e7.png">

Reply:
Må vi også lage en binding til `resultIndicators` her da, for å lytte på endringer der?

Reply:
Har vi ikke det, utenfra komponenten, i store?